include/stl2/view/subrange.hpp

```diff
@@ -216,6 +241,9 @@ STL2_OPEN_NAMESPACE {
 		}
 	};
 
+	template<Iterator I, Sentinel<I> S>
+	subrange(I, S) -> subrange<I, S>;
+
 	template<Iterator I, Sentinel<I> S>
```

Comment:
Ditto `Sentinel` is redundant. We do need `WeaklyIncrementable` for `iter_difference_t`, but `Iterator` is probably less confusing.
```suggestion
	template<Iterator I, class S>
```